Checkers/Checker.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+
+namespace Checkers
+{
+
+    public class Checker
+    {
+        public string Symbol  { get; set; }
+        public int[] Position  { get; set; }
+        public string Color { get; set; }
+        
+        public Checker(string color, int[] position)
+        {
+            this.Color = color.ToLower();
+            this.Position = position;
+
+            if(this.Color != null || this.Color != "")
+            {
+                if(this.Color == "white")
+                {
+                    createWhiteChecker();
+
+                }
+
+                else
+                {
+                    createBlackChecker();
+
+                }
+            }
+        }
+
+        //generates the desired symbol using utf32
+        private string whiteChecker()
+        {
+            int openCircleId = int.Parse("25CB", System.Globalization.NumberStyles.HexNumber);
+            string openCircle = char.ConvertFromUtf32(openCircleId);
+            return openCircle;
+
+        }
+
+            //generates the desired symbol using utf32
+        private string blackChecker()
+        {
+            int closedCircleId = int.Parse("25CF", System.Globalization.NumberStyles.HexNumber);
+            string closedCircle = char.ConvertFromUtf32(closedCircleId);
+            return closedCircle;
+
+        }
+
+        // sets the symbol for the checker instance
+        public void createWhiteChecker()
```

Comment:
You could move this logic into the setter for the `Color` property then make the Symbol property a get only. Just a thought to clean this up a bit, won't affect the outcome or the program or your grade.